Detectors/CPV/reconstruction/src/RawDecoder.cxx

```diff
@@ -63,14 +63,16 @@ RawErrorType_t RawDecoder::readChannels()
       }
       skipUntilNextHeader = false;
       currentBC = header.bc();
+      currentOrbit = header.orbit();
```

Comment:
Hi @peressounko Here currentOrbit = mRawReader.getCurrentHBFOrbit() is orbit we read from RDH (see line 52). 
header.orbit() is orbit read from cpv header. In the lines 68-73 I try to compare orbit recorded in RDH with orbit recorded in cpv payload. The added line currentOrbit = header.orbit() will just void this comparison, and we will never know if cpv payload is self-consistent with its RDH.

Reply:
agree. If we revert check of orbit consistency in RDH and trigger header, this should be reverted too.